### PR TITLE
Add goal Q1 2024 to usage/total charts

### PR DESCRIPTION
**Pull request by majamassarini**
Fixes packit/packit-service#2304

Merge after packit/packit-service#2330

### CODE REVIEW

**Comment by majamassarini** [comment]
recheck

**Review by lachmanfrantisek — COMMENTED** [review]
Thanks a lot, Maja! When you are in this, can you please remove the chart from the last quarter?

**Review by lbarcziova — APPROVED** [review]
:+1: 

do you think we should also show the almost_onboarded part? :thinking:

File: frontend/src/app/Usage/UsageList.tsx [hunk 1]
```diff
@@ -254,6 +254,45 @@ const UsageList: React.FC<UsageListProps> = (props) => {
...
+                  ariaDesc="Chart describing downstream onboarding."
+                  ariaTitle="Chart describing downstream onboarding."
+                  constrainToVisibleArea
+                  labels={({ datum }) => `${datum.name}: ${datum.y}`}
+                  maxDomain={{ y: 500 }}
+                  name="Chart describing downstream onboarding."
+                  primarySegmentedMeasureData={[
+                    {
+                      name: "Onboarded projects",
+                      y: data.onboarded_projects_q1_2024,
```
Comment (lbarcziova) [inline frontend/src/app/Usage/UsageList.tsx:281]: actually, where is this field coming from please?

**Comment by majamassarini** [comment]
> 👍
> 
> do you think we should also show the almost_onboarded part? 🤔

Do you mean the number of the almost_onboarded projects or their url? 
I think the useful part is their url, so we can maybe do something to push them forward, but I don't think this view is the right place where to have this list. And it is so easy to get it via the usage page (for us) that I would not spend time on it. But probably this is also because I don't know React so well to do it quickly...

File: frontend/src/app/Usage/UsageList.tsx (hunk 1 above)
Comment (majamassarini) [reply frontend/src/app/Usage/UsageList.tsx:281]: https://github.com/packit/packit-service/blob/48e820856a32aa1f997324bafcec3eec865f67a3/packit_service/service/api/usage.py#L738

Comment (lbarcziova) [reply frontend/src/app/Usage/UsageList.tsx:281]: sorry it looks like I am blind :D thanks for the link!

**Pull request merged**